jamaljsr/polar
# feat: make lightning nodes p2p port reachable on LAN

Closes #298 

### Description

This commit changes lightning nodes to accept p2p connections via LAN instead of only via the private network (172.x.x.x).

As with #296, this is also a breaking change that needs migration.

### Steps to Test

1. Add a new network with a lnd node and a c-lightning node
2. Add a lnd and a c-lightning node from the Designer
3. Check the connectivity tab for each lightning node
4. Try to connect to the lightning nodes from within the LAN

### Screenshots

![P2P on ConnectTab](https://user-images.githubusercontent.com/3824379/75048762-59349780-54c9-11ea-8c14-b85225307e8e.png)
(IP address for the P2P URI has changed from `172.x.x.x` to `127.0.0.1`)
